spec/controllers/articles_controller_spec.rb

```diff
@@ -0,0 +1,124 @@
+require 'spec_helper'
+
+RSpec.describe ArticlesController do
+  describe 'GET index' do
+    describe '.html' do
+      it 'renders the index template and sets @articles' do
+        get :index
+        expect(assigns(:articles)).to eq([])
+        expect(response.status).to eq(200)
+        expect(response).to render_template("index")
+      end
+
+      describe 'with 20 articles' do
+        before(:each) { 20.times { create(:article, published: true) } }
+
+        it 'returns 10 articles in created at order' do
+          expected = Article.where(page: 1)
+                              .order(created_at: :desc)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.